int32 left_enc
int32 right_enc
int16 vel_v
int16 vel_w
int16 adc
int64 time
uint16 rate
